modules/developers-guide/pages/tutorials/custom-frontend.adoc

```diff
@@ -336,19 +316,7 @@ For example, you might want to change the font family and use a placeholder for
 ....
 include::example$react-revised-index.jsx[]
```

Comment:
A bit of a nit here, but the code in this snippet doesn't include the jsx imports. If I'm being lazy / sloppy and simply copying and pasting from the snippet and over writing everything in my index.jsx I will no longer have 

`import { custom_greeting } from "../../declarations/custom_greeting";

// Insert these lines after the import statement on line 1
import * as React from "react";
import { render } from "react-dom";`

Not a major thing but might break some newbs